app/src/main/java/edu/mit/mitmobile2/MITActivity.java

```diff
@@ -26,7 +26,7 @@
         GoogleApiClient.OnConnectionFailedListener,
         LocationListener {
 
-    private static final long UPDATE_INTERVAL = 120000;
+    private static final long UPDATE_INTERVAL = 10000;
```

Comment:
We really don't need to be checking the location every 10 seconds - if 2 mins is too infrequent, just cut it to 1 min